pkgs/development/tools/gomod2nix/update.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i python3 -p nix -p python3 -p python3.pkgs.requests
+from pprint import pprint
+import subprocess
+import requests
+import os.path
+import shutil
+import json
+
+
+OWNER = "tweag"
+REPO = "gomod2nix"
+URL = f"https://api.github.com/repos/{OWNER}/{REPO}/releases/latest"
+
+
+# Ideally this would use shutil.copytree, but that doesn't give control over directory permissions
+def copy_tree(src, dst):
+    """Copy a tree without permissions"""
+    subprocess.check_output([
+        "cp",
+        "-r",
+        "--no-preserve=mode,ownership",
+        src,
+        dst,
+    ])
+
+
+if __name__ == "__main__":
+
+    pkg_dir = os.path.dirname(os.path.abspath(__file__))
+    nixpkgs_dir = os.path.abspath(os.path.join(pkg_dir, "../../../../"))
+
+    builder_dir = os.path.join(nixpkgs_dir, "pkgs/build-support/go/gomod2nix")
```

Comment:
if the nixpkgs root dir is relative then we must already be in this directory, right? Then this could be derived from PWD/CWD and when moving this we don't need to remember to also update the path here.

Reply:
The script has been ported to bash.